package.json

```diff
@@ -1,42 +1,59 @@
 {
   "name": "scratchblocks",
-  "version": "3.1.2",
   "description": "Render scratchblocks code to SVG images.",
-  "browser": "browser.js",
-  "directories": {
-    "test": "tests"
-  },
-  "prettier": {
-    "semi": false,
-    "trailingComma": "es5"
-  },
-  "scripts": {
-    "test": "jest",
-    "build": "browserify browser.js -o build/scratchblocks.min.js -t [ babelify --presets [ es2015 babili ] ]",
-    "fmt": "prettier --write *.js syntax/*.js scratch2/*.js",
-    "start": "beefy browser.js:bundle.js --live --port 8000"
-  },
+  "license": "MIT",
+  "author": "tjvr",
+  "homepage": "https://scratchblocks.github.io/",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/tjvr/scratchblocks.git"
   },
-  "keywords": [
-    "scratch"
-  ],
-  "author": "tjvr",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/tjvr/scratchblocks/issues"
   },
-  "homepage": "https://scratchblocks.github.io/",
-  "devDependencies": {
+  "version": "3.1.2",
+  "main": "build/scratchblocks.min.js",
+  "browser": "build/scratchblocks.min.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "browserify browser.js -o build/scratchblocks.min.js -t [ babelify --presets [ es2015 babili ] ]",
+    "fmt": "prettier --write *.js syntax/*.js scratch2/*.js",
+    "postinstall": "npm run build -- -s Scratchblocks",
+    "lint:staged": "lint-staged",
+    "start": "beefy browser.js:bundle.js --live --port 8000",
+    "test": "jest"
```

Comment:
How does this work, since `build/` isn't committed? :)

Reply:
The `build` here refers to the npm script, not the directory. So it's just like when you run `npm run build` in Terminal. I've actually got another branch in the works that updates the config for building stuff a little, and that DOES include the build directory in the tree, so we can chat about that then.

For now if you want to see this in action, run `npm i` in the root of the repo and you'll see the postinstall command run, and the `scratchblocks.min.js` file created in the `build` directory :)

Reply:
I've been testing this on my local copy of our projects site, where `import scratchblocks from 'scratchblocks'` works fine, but I'll confirm that `require('scratchblocks')` also still works. Can't see why it wouldn't, but will make absolutely sure anyway :)

Reply:
Ah, I missed the `postinstall` command! Gotcha. :)